### PR TITLE
Refactor credential text, add context and creation time

### PR DESCRIPTION
**Pull request by richardhuaaa**
Still to-do:
- Use `MlsCredential` wrapper proto
- Check/update validation logic, e.g. verifying installation public key matches leaf node
- Use `account_address` instead of `wallet_address` everywhere

### CODE REVIEW

**Comment by neekolas** [comment]
@richardhuaaa can we get this merged today? I've removed the `xmtp` crate in another PR and fixed up a couple of lint warnings. Looks pretty good to go, although we might want to converge around the name `account_address`. I see a mix of `blockchain_address` and `wallet_address` in different places. Could be done as a subsequent PR.

File: xmtp_mls/IDENTITY.md
Comment (richardhuaaa) [inline xmtp_mls/IDENTITY.md:58]: @jhaaaa @saulmc - would love your feedback on the verbiage in the signature text. Q's:

- Should we add text explaining what is being granted, what the installation ID is used for, what kind of verification the user needs to do? We need the user to verify the 'current time' is correct, and that they are signing for the correct 'account address'. The installation ID is not a secret and they don't need to memorize it. The 'installation ID' and 'current time' are there so they can list installations in the future and recognize this one, revoking it if needed.
- Should we change the labels for clarity, for example `Granted Time` instead of `Current Time`, `Installation Token` or `Granted Token` etc instead of `Installation Id`?
- Ethereum address are formatted as `0x<random string>`, should installation Id's be formatted differently for clarity, e.g. `x:<Random String>`?

**Review by neekolas — APPROVED** [review]

File: xmtp_mls/IDENTITY.md
Comment (richardhuaaa) [inline xmtp_mls/IDENTITY.md:56]: One thing I just realized, this leaks the user's timezone. Curious on other's thoughts here

Comment (neekolas) [reply xmtp_mls/IDENTITY.md:56]: Good catch. Any harm in converting to UTC? Guess it makes it a little less human readable, but I can't imagine people are going to be reading these timestamps too closely

Comment (richardhuaaa) [reply xmtp_mls/IDENTITY.md:56]: Then we can't really ask users to double-check the timestamp, and we're trusting the app to furnish a valid timestamp. I suppose it's okay if a malicious app forges a fake timestamp here, as it's only used for identification? I can change it for now

File: xmtp_mls/IDENTITY.md
Comment (richardhuaaa) [reply xmtp_mls/IDENTITY.md]: Merging this to unblock others, but will keep track of this thread and add changes separately!

Comment (saulmc) [reply xmtp_mls/IDENTITY.md]: Defer to @jhaaaa 's expertise on these questions!

**Pull request merged**
